app/static/app/js/components/Map.jsx

```diff
@@ -7,6 +7,8 @@ import Leaflet from 'leaflet';
 import async from 'async';
 import 'leaflet-measure/dist/leaflet-measure.css';
 import 'leaflet-measure/dist/leaflet-measure';
+import 'leaflet-draw/dist/leaflet.draw.css';
+import 'leaflet-draw/dist/leaflet.draw';
```

Comment:
You might need to add `leaflet-draw` in `package.json` as well!

Reply:
Yes, of course